client/jetpack-connect/user-type/style.scss

```diff
@@ -0,0 +1,59 @@
+.user-type__wrapper {
```

Comment:
I see a bunch of duplication here. I thought we were going to componentize this?